src/app/@ansyn/imagery/model/visualizers/initial-annotations-feature-properties.ts

```diff
@@ -0,0 +1,42 @@
+import { IVisualizerStyle, MarkerSize } from "./visualizer-style"
+import { cloneDeep } from "lodash";
+
+const ANNOTATIONS_INITIAL_STYLE: IVisualizerStyle = {
```

Comment:
i think the the "initial" style should come from config - in case it's not in config then use this hard coded version